esmvaltool/diag_scripts/example_diagnostic.py

```diff
@@ -0,0 +1,46 @@
+"""Python example diagnostic."""
```

Comment:
Should we put this in a subdirectory `diag_scripts/examples/` together with the `mydiag` scripts?